### PR TITLE
Ability to handle multi dataPoints via handleSelection

### PR DESCRIPTION
**Pull request by ignatvilesov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 3.1.1
```
Comment (EugeneElkin) [inline CHANGELOG.md]: 3.2.0 is better because it looks like a new feature

File: lib/index.d.ts [hunk 2]
```diff
@@ -151,7 +151,7 @@ declare module powerbi.extensibility.utils.interactivity {
          * identity is undefined, the selection state is cleared. In this case, if specificIdentity
          * exists, it will still be sent to the host.
          */
-        handleSelection(dataPoint: SelectableDataPoint, multiSelect: boolean): void;
+        handleSelection(dataPoints: SelectableDataPoint | SelectableDataPoint[], multiSelect: boolean): void;
```
Comment (EugeneElkin) [inline lib/index.d.ts]: To reduce number of fixes in visuals I reccoment to make multiSelect parameter as nullable

Comment (ignatvilesov) [reply lib/index.d.ts]: The `multiSelect` has not been changed. The interface looks the same. In other words, there's no need to change CVs

File: CHANGELOG.md (hunk 1 above)
Comment (ignatvilesov) [reply CHANGELOG.md]: Agree. Will update

Comment (ignatvilesov) [reply CHANGELOG.md]: Done

**Pull request merged**